### PR TITLE
Update Flagsmith integration instructions

### PR DESCRIPTION
**Pull request by dabeeeenster**
### What does this PR do?

Updates the Flagsmith installation instructions to correct them.

### Motivation

A fix was needed!

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

N/A

### CODE REVIEW

File: flagsmith/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ All flag change events are sent to Datadog. These events are tagged with the env
 
 ## Setup
 
-In the [Flagsmith Integration tile][2], enter your [Datadog API Key][3]. For API URL, enter `https://api.datadoghq.com` if you are using the US site, or `https://api.datadoghq.eu` if you are using the EU site.
+In the [Flagsmith Dashboard](https://app.flagsmith.com/), select the Integrations Menu and then add the Datadog Integration. Enter your [Datadog API Key][2]. For Base URL, enter `https://api.datadoghq.com` if you are using the US Datadog site, or `https://api.datadoghq.eu` if you are using the EU Datadog site.
```
Comment (jtappa) [inline flagsmith/README.md:11]:
```suggestion
In the [Flagsmith Dashboard][2], select the Integrations Menu and then add the Datadog Integration. Enter your [Datadog API Key][3]. For Base URL, enter `https://api.datadoghq.com` if you are using the US Datadog site, or `https://api.datadoghq.eu` if you are using the EU Datadog site.
```

File: flagsmith/README.md [hunk 2]
```diff
@@ -26,10 +26,9 @@ All Flagsmith events are sent to the Datadog event stream.
...
+Need help? See the [Flagsmith documentation][3] or contact [Datadog Support][4].
 
 [1]: https://www.flagsmith.com/
-[2]: https://app.datadoghq.com/account/settings#integrations/flagsmith
-[3]: https://app.datadoghq.com/organization-settings/api-keys
-[4]: https://docs.flagsmith.com/integrations/datadog/
-[5]: https://docs.datadoghq.com/help/
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+[3]: https://docs.flagsmith.com/integrations/datadog/
+[4]: https://docs.datadoghq.com/help/
```
Comment (jtappa) [inline flagsmith/README.md:34]:
```suggestion
[2]: https://app.flagsmith.com/
[3]: https://app.datadoghq.com/organization-settings/api-keys
[4]: https://docs.flagsmith.com/integrations/datadog/
[5]: https://docs.datadoghq.com/help/
```

File: flagsmith/README.md [hunk 3]
```diff
@@ -26,10 +26,9 @@ All Flagsmith events are sent to the Datadog event stream.
 
 ## Troubleshooting
 
-Need help? See the [Flagsmith documentation][4] or contact [Datadog Support][5].
+Need help? See the [Flagsmith documentation][3] or contact [Datadog Support][4].
```
Comment (jtappa) [inline flagsmith/README.md:29]:
```suggestion
Need help? See the [Flagsmith documentation][4] or contact [Datadog Support][5].
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Comment by dabeeeenster** [comment]
OK thanks all suggestions committed!

**Review by jtappa — APPROVED** [review]

**Pull request merged**